### PR TITLE
Fix go PostCommit break

### PR DESCRIPTION
**Pull request by lostluck**
I committed 3 errors here:
* The test wasn't covering what I thought it was, since none of the TestInvoke functions have shims.
* The breaking error: The type assertion to error on a nil, which isn't permitted. Needs a check first.
* The event time (which is currently sporadically used), wasn't being checked properly, the wrong index was being used.

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/) [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by lostluck** [comment]
R: @apilloud

File: sdks/go/pkg/beam/core/runtime/exec/fn_arity.tmpl [hunk 1]
```diff
@@ -49,22 +51,19 @@ func (n *invoker) initCall() {
 
             // (5) Return direct output, if any. Input timestamp and windows are implicitly
```
Comment (apilloud) [inline sdks/go/pkg/beam/core/runtime/exec/fn_arity.tmpl]: This file has a weird mix of tabs and spaces.

Comment (lostluck) [reply sdks/go/pkg/beam/core/runtime/exec/fn_arity.tmpl]: Done.

**Comment by lostluck** [comment]
Missed a rename in the template.

**Comment by apilloud** [comment]
LGTM

**Pull request merged**